pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -323,6 +325,15 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
         return _getNormalizedWeights(tokens);
     }
 
+    /**
+     * @dev Returns the normalized weight of a single token.
+     */
+    function _getNormalizedWeight(IERC20 token) internal view returns (uint256) {
+        uint256 weightChangeProgress = ManagedPoolStorageLib.getGradualWeightChangeProgress(_poolState);
+
+        return ManagedPoolTokenLib.getTokenWeight(_tokenState[token], weightChangeProgress);
+    }
```

Comment:
I think if we're only using this getter in one place then we can just inline it.

Reply:
I was trying to get rid of it, but it is used in two places.